[2024-05-14T08:13:40.235+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T08:13:40.244+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T08:13:30.035048+00:00 [queued]>
[2024-05-14T08:13:40.247+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T08:13:30.035048+00:00 [queued]>
[2024-05-14T08:13:40.247+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T08:13:40.253+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_lake> on 2024-05-14 08:13:30.035048+00:00
[2024-05-14T08:13:40.256+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=64) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T08:13:40.256+0000] {standard_task_runner.py:63} INFO - Started process 66 to run task
[2024-05-14T08:13:40.257+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'data_lake', 'manual__2024-05-14T08:13:30.035048+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmpz2a24pbo']
[2024-05-14T08:13:40.258+0000] {standard_task_runner.py:91} INFO - Job 78: Subtask data_lake
[2024-05-14T08:13:40.276+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.data_lake manual__2024-05-14T08:13:30.035048+00:00 [running]> on host 2d25b1319046
[2024-05-14T08:13:40.313+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='data_lake' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T08:13:30.035048+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T08:13:30.035048+00:00'
[2024-05-14T08:13:40.314+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T08:13:40.320+0000] {logging_mixin.py:188} INFO - data_lake !!!
[2024-05-14T08:13:40.320+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T08:13:40.321+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T08:13:40.325+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=data_lake, run_id=manual__2024-05-14T08:13:30.035048+00:00, execution_date=20240514T081330, start_date=20240514T081340, end_date=20240514T081340
[2024-05-14T08:13:40.357+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T08:13:40.369+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T08:13:40.370+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
